core/indigo-core/reaction/src/reaction_json_loader.cpp

```diff
@@ -32,7 +34,32 @@ using namespace rapidjson;
 
 IMPL_ERROR(ReactionJsonLoader, "reaction KET loader");
 
-ReactionJsonLoader::ReactionJsonLoader(Document& ket) : _rgroups(kArrayType), _molecule(kArrayType), _pluses(kArrayType), _arrows(kArrayType)
+void ReactionJsonLoader::getMoleculeBoundingBox(BaseMolecule& mol, Rect2f& bbox)
```

Comment:
It seems can be declared static (unrelated to an instance)

Reply:
method moved to BaseMolecule